OpenEdXMobile/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -46,7 +50,8 @@ public CoursesAsyncLoader(Context context) {
 
         try {
             if (profile != null) {
-                enrolledCoursesResponse = api.getUserEnrolledCourses(profile.username, false);
+                String org = config.getOrganizationCode();
```

Comment:
Maybe change the variable name from `org` to `orgCode` to make it consistent with the naming in `RestApiManager`?